CMakeLists.txt

```diff
@@ -56,7 +59,7 @@ set(CMAKE_CXX_STANDARD 17)
 
 project(VBA-M C CXX)
 
-find_package(Git)
+find_package(Git REQUIRED)
```

Comment:
If git is not actually required for linux tarball builds, then I wouldn't make it be, but I will check this myself.

Reply:
We depend on git for a lot of things and fail in various places if we don't. I think it's better to fail explicitly early in the build process.

Reply:
We have to support building a tarball without git like any proper linux software. So if git is being used somewhere in that process, it needs to skip that part for that particular use-case. We don't have to support not having git for e.g. `-DUPSTREAM_RELEASE=TRUE` because that's our stuff and not for people to use.

Reply:
I reverted this and cleaned up some more stuff.